### PR TITLE
Add Spanish translations

### PR DESCRIPTION
**Pull request by kbravh**
I've added a Spanish translation for almost all files except the keyboards.md. 

I also added a translation for the license.md file. The translations are not official translations of the licenses, but may serve to let Spanish speakers better understand what the licenses consist of.

I added a disclaimer at the top of the page saying they aren't official translations and advise the user to refer to the official licenses in English with a link to the English license page.

Is this the route we want to take for the licenses? I can remove the translation altogether to fallback to the English license page.

### CODE REVIEW

File: _i18n/es/changes.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: "Cambios recientes en QMK"
+layout: qmk-title
+lang: "es"
+---
+
+{% for change in site.changes %}
```
Comment (kuchosauronad0) [inline _i18n/es/changes.md]:
```suggestion
{% for change in site.changes reversed %}
```
This way the most recent change is at the top of the page. See #78

**Comment by noroadsleft** [comment]
Merged updated branch to fix a merge conflict (lol).

@namelivia do you mind reviewing this for me?

File: _i18n/es/coc.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: "QMK Código de conducta"
+layout: qmk
+lang: "es"
```
Comment (noroadsleft) [inline _i18n/es/coc.md:5]:
```suggestion
permalink: /es/coc/
lang: "es"
```

File: _i18n/es/converter.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Convertir KLE a QMK info.json"
+layout: qmk
+lang: "es"
```
Comment (noroadsleft) [inline _i18n/es/converter.md:5]:
```suggestion
permalink: /es/converter/
lang: "es"
```

File: _i18n/es/flasher.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+layout: qmk
+lang: "es"
```
Comment (noroadsleft) [inline _i18n/es/flasher.md:4]:
```suggestion
permalink: /es/flasher/
lang: "es"
```

... Though I'm not actually sure this page is used anymore.

File: _i18n/es/license.md [hunk 5]
```diff
@@ -0,0 +1,472 @@
+---
+title: "Licencia"
+layout: qmk
+lang: "es"
```
Comment (noroadsleft) [inline _i18n/es/license.md:5]:
```suggestion
permalink: /es/license/
lang: "es"
```

File: _i18n/es/powered.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
+---
+title: "Usar la marca registrada del Firmware QMK"
+layout: qmk
+lang: "es"
```
Comment (noroadsleft) [inline _i18n/es/powered.md:5]:
```suggestion
permalink: /es/powered/
lang: "es"
```

File: _i18n/es/proton-c.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Sobre el Proton C"
+layout: qmk
+lang: "es"
```
Comment (noroadsleft) [inline _i18n/es/proton-c.md:5]:
```suggestion
permalink: /es/proton-c/
lang: "es"
```

File: _i18n/es/support.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
+---
+title: "Soporte de QMK"
+layout: qmk
+lang: "es"
```
Comment (noroadsleft) [inline _i18n/es/support.md:5]:
```suggestion
permalink: /es/support/
lang: "es"
```

File: _i18n/es/toolbox.md [hunk 9]
```diff
@@ -0,0 +1,26 @@
+---
+title: "QMK Toolbox"
+layout: qmk-editable
+lang: "es"
```
Comment (noroadsleft) [inline _i18n/es/toolbox.md:5]:
```suggestion
permalink: /es/toolbox/
lang: "es"
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Comment by namelivia** [comment]
@noroadsleft Could you create a review request for me user so I don't forget about this? Thanks

**Comment by noroadsleft** [comment]
@namelivia I actually can't; GitHub won't let me review request anyone who isn't a Member of QMK. I'm not sure if this is a GitHub limitation, or a limitation because the repository is owned by an organization rather than a user. (I've wanted to do this on other QMK repos too, and I can't do it there either.)

If you can't get to it, it's okay. Anything broken or wrong can be fixed later.

**Comment by kbravh** [comment]
Alright, I've added the permalinks!

**Review by noroadsleft — APPROVED** [review]
Looks good, thanks!

**Pull request merged**